SU2_CFD/src/iteration_structure.cpp

```diff
@@ -494,95 +494,46 @@ void CMeanFlowIteration::Iterate(COutput *output,
                                  CFreeFormDefBox*** FFDBox,
```

Comment:
Ok! That's what I thought. As I have mentioned before, for multiphysics applications the coupling is normally done after convergence of each field, but I do see the point of this for multi-zone fluid problems. Thanks!